packages/cozy-client/src/CozyClient.spec.js

```diff
@@ -224,8 +224,11 @@ describe('CozyClient', () => {
 
     it('should dehydrate relationships', async () => {
       class FakeHasMany extends Association {
-        get raw() {
-          return this.target[this.name]
+        dehydrate(doc) {
+          return {
+            ...doc,
+            [this.name]: this.target[this.name]
+          }
```

Comment:
Because this test runs on a mock association. Before this update it had the `raw` logic here, which is effectively the same.

I chose to keep this test, not because it does a good job of testing the actual dehydratation logic (the new `helpers.spec.js` does that), but because it tests that documents passed to `client.save()` get dehydrated.